README.md

```diff
@@ -1,49 +1,100 @@
-# Fresh SEO 🍋
 
-The fastest way ⚡️ to create sitemaps in your Deno
-[Fresh project](https://fresh.deno.dev/) project 🍋
+# Fresh SEO 🍋   [![Badge License]][License]
 
-## Getting started
+*The fastest way to create sitemaps* <br>
+*for your Deno **[Fresh project]**.*
 
-To get started, run the init command at the root of your project.
+<br>
+
+## Getting Started
+
+*Run the setup at the root of your project.*
+
+```shell
+deno run            \
+    --allow-read    \
+    --allow-write   \
+    https://deno.land/x/fresh_seo/init.ts
 
-```bash
-deno run --allow-read --allow-write https://deno.land/x/fresh_seo/init.ts
 ```
 
-A new file `./routes/sitemap.xml.ts` should have been created.
+<br>
+
+The following file should have been created:
+
+`routes/sitemap.xml.ts`
+
+<br>
 
-A basic `sitemap.xml` should be available at
-`http://localhost:8000/sitemap.xml`.
+A basic sitemap should now be available at:
+    
+[`http://localhost:8000/sitemap.xml`][Localhost]
+    
+<br>
+<br>
 
 ## How does it work?
 
-Fresh SEO automatically maps out **static** routes in your Fresh project. For basic routes, 
-you do not have to do anything manually.
+**Fresh SEO 🍋** automatically maps <br>
+out **static** routes in your project.
 
-However, you will still have to map dynamic routes yourself!
+*For basic routes, you do not* <br>
+*have to do anything manually.*
 
-```diff
+<br>
+
+### Dynamic Routes
+
+*You will still have to map dynamic routes yourself!*
+
+```js
 // ./routes/sitemap.xml.ts
-import { Handlers } from "$fresh/server.ts";
-import manifest from "../fresh.gen.ts";
-import { SitemapContext } from "https://deno.land/x/fresh_seo/mod.ts";
-
-export const handler: Handlers = {
-  GET(req, ctx) {
-    const sitemap = new SitemapContext(
-      "http://example.com"
-      manifest
-    );
-
-    // You can add additional page here
-+   sitemap.add("/blog/hello-world");
-
-    return sitemap.render();
-  },
-};
+import { SitemapContext } from 'https://deno.land/x/fresh_seo/mod.ts';
+import { Handlers } from '$fresh/server.ts';
+import manifest from '../fresh.gen.ts';
+
+export const handler : Handlers = {
+    
+    GET(request,context){
+        
+        const sitemap = new SitemapContext(
+            'http://example.com'
+            manifest
+        );
+```
+
+```js
+        // You can add additional page here
+        sitemap.add('/blog/hello-world');
 ```
 
+```js
+        return sitemap.render();
+    }
+}
+```
```

Comment:
There seems to be a confusion here as this code should be in one snippet

Reply:
> Diff ➞ JS
>
> Indicated addition through separated sections.
>
> Syntax coloring is more fresh lemon

Instead of using a diff block where only the addition has any sort of highlighting / coloring, I used separated js blocks, this way the added section is still differrenciable via it being a separate block but at the same time the code is colored in.

Reply:
Oh, I see, I guess we could just remove the diffing and put all in one block then. I didn't realize the coloring wasn't working, but as we have comment, it should be fine to remove the diff.

Reply:
So you are fine with my design choice?